source/install/cloud-changelog.md

```diff
@@ -4,12 +4,53 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-12-01](#release-2022-12-01)
 - [Release 2022-11-24](#release-2022-11-24)
 - [Release 2022-11-17](#release-2022-11-17)
 - [Release 2022-11-10](#release-2022-11-10)
 - [Release 2022-10-27](#release-2022-10-27)
 - [Release 2022-10-20](#release-2022-10-20)
-- [Release 2022-10-13](#release-2022-10-13)
+
+## Release 2022-12-01
+
+### Improvements
+
+#### User Interface (UI)
+ - Added message priority labels to the Threads view.
+ - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts.
+ - Added a centralized page for draft messages.
```

Comment:
I can remove these if we're planning to keep the feature flags disabled for this cloud release.

Reply:
Thanks @amyblais, we should plan to enable the feature flag for drafts and post priority for this cloud release (pending community testing and QA release approval)